src/autopas/pairwiseFunctors/CellFunctor.h

```diff
@@ -132,9 +138,19 @@ template <class Particle, class ParticleCell, class ParticleFunctor, DataLayoutO
 void CellFunctor<Particle, ParticleCell, ParticleFunctor, DataLayout, useNewton3, bidirectional>::processCellPair(
 
     ParticleCell &cell1, ParticleCell &cell2, const std::array<double, 3> &sortingDirection) {
-  if ((DataLayout == DataLayoutOption::soa && (cell1._particleSoABuffer.getNumberOfParticles() == 0 ||
-                                               cell2._particleSoABuffer.getNumberOfParticles() == 0)) ||
-      (DataLayout == DataLayoutOption::aos && (cell1.numParticles() == 0 || cell2.numParticles() == 0))) {
+  if ((DataLayout == DataLayoutOption::soa &&
+       (cell1._particleSoABuffer.size() == 0 and cell2._particleSoABuffer.size() == 0)) or
+      (DataLayout == DataLayoutOption::aos && (cell1.size() == 0 and cell2.size() == 0))) {
+    return;
```

Comment:
```suggestion
  if ((DataLayout == DataLayoutOption::soa and
       (cell1._particleSoABuffer.size() == 0 and cell2._particleSoABuffer.size() == 0)) or
      (DataLayout == DataLayoutOption::aos and (cell1.size() == 0 and cell2.size() == 0))) {
    return;
```

Reply:
done in [06385c2](https://github.com/AutoPas/AutoPas/pull/782/commits/06385c264cc416ebc407bb395db2e8d1a70c1cd0)